src/lesson-3/modules/tasks/task.controller.ts

```diff
@@ -0,0 +1,44 @@
+import { TaskService } from './task.service';
+import {
+  FilterParams,
+  NewTaskBody,
+  TaskDetails,
+  TaskType,
+  UnitedSchema,
+} from './task.types';
+
+export class TaskController {
+  taskService: TaskService;
+
+  constructor(tasks: UnitedSchema[]) {
```

Comment:
service краще передавати через конструктор, не створювати в самому класі (dependency injection). таким чином клас не буде залежати від сервісу напряму
```suggestion
  constructor(service: TaskService) {
  
  new TaskController(new TaskService(tasks))
```